website/source/docs/guides/autopilot.html.markdown

```diff
@@ -0,0 +1,108 @@
+---
+layout: "docs"
+page_title: "Autopilot"
+sidebar_current: "docs-guides-autopilot"
+description: |-
+  This guide covers how to configure and use Autopilot features.
+---
+
+# Autopilot
+
+Autopilot is a set of new features added in Consul 0.8 to allow for automatic
+operator-friendly management of Consul servers. It includes cleanup of dead
+servers, monitoring the of the Raft cluster, and stable server introduction.
+
+To enable Autopilot features (with the exception of dead server cleanup),
+the [`raft_protocol`](/docs/agent/options.html#_raft_protocol) setting in
+the Agent configuration must be set to 3 or higher on all servers. In Consul
+0.8 this setting defaults to 2; in Consul 0.9 it will default to 3.
+
+## Configuration
+
+The configuration of Autopilot is loaded by the leader from the agent's
+[`autopilot`](/docs/agent/options.html#autopilot) settings when initially
+bootstrapping the cluster. After bootstrapping, the configuration can
+be viewed or modified either via the [`operator autopilot`]
+(/docs/commands/operator/autopilot.html) subcommand or the
+[`/v1/operator/autopilot/configuration`](/docs/agent/http/operator.html#autopilot-configuration)
+HTTP endpoint:
+
+```
+$ consul operator autopilot get-config
+CleanupDeadServers = true
+LastContactThreshold = 200ms
+MaxTrailingLogs = 250
+ServerStabilizationTime = 10s
+
+$ consul operator autopilot set-config -cleanup-dead-servers=false
+Configuration updated!
+
+$ consul operator autopilot get-config
+CleanupDeadServers = false
+LastContactThreshold = 200ms
+MaxTrailingLogs = 250
+ServerStabilizationTime = 10s
+```
+
+## Dead Server Cleanup
+
+Dead servers will periodically be cleaned up and removed from the Raft peer
+set, to prevent them from interfering with the quorum size and leader elections.
+This cleanup will also happen whenever a new server is successfully added to the
+cluster.
```

Comment:
I'd add:

```
Prior to Autopilot, it would take 72 hours for dead servers to be automatically reaped,
or operators had to script a `consul force-leave`. If another server failure occurred,
it could jeopardize the quorum, even if the failed Consul server had been automatically
replaced. Autopilot helps prevent these kinds of outages by quickly removing failed
servers as soon as a replacement Consul server comes online.
```